### PR TITLE
Fix pbrt exporter coordinate system and FoV

### PR DESCRIPTION
**Pull request by skogler**
Hey, 

this is what I had to do to get the PBRT exporter to correctly convert from GLTF2 to PBRT scenes for my use case.

Let me know what you think!

### CODE REVIEW

File: code/Pbrt/PbrtExporter.cpp [hunk 1]
```diff
@@ -111,7 +111,22 @@ PbrtExporter::PbrtExporter(
...
+            0.f,  1.f,  0.f, 0.f, //
+            0.f,  0.f,  0.f, 1.f  //
+        ) {
+
+    mRootTransform = aiMatrix4x4(
+        -1.f,  0,  0.f, 0.f, //
+        0.0f,  -1.f,  0.f, 0.f, //
+        0.f,  0.f,  1.f, 0.f, //
+        0.f,  0.f,  0.f, 1.f  //
+    ) * mRootTransform;
```
Comment (skogler) [inline code/Pbrt/PbrtExporter.cpp:129]: Was not sure why I need to rotate 180 around the Z axis here, but it seems to work.

**Review by kimkulling — APPROVED** [review]
Seems to work, lets try it out.

**Review by kimkulling — APPROVED** [review]
Seems to work, lets try it out.

File: code/Pbrt/PbrtExporter.cpp (hunk 1 above)
Comment (kimkulling) [reply code/Pbrt/PbrtExporter.cpp:129]: Honestly I am not sure as well. Let's give it a try.

**Comment by kimkulling** [comment]
Merged, thanks a lot for your contribution.

**Pull request merged**